### DISCUSSION TITLE
[Weekly Meeting] [2024-09-27] Meeting Minutes

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#1499] [Weekly Meetings]
First batch of Images being merged into KIE Tools
- thank you @ricardozanini !

Apache KIE 10 release
- Found problem to build from source
  - PR raised and merged.
  - New build and new images uploaded
- Still strugling to build from source
  -  @porcelli in contact with @ricardozanini to verify
- Some NOTICE files and licenses are missing
  - most isolated to -test jars (might not be a blocker)
  - Optaplanner docs missing license
  - Images missing licenses (as binary might not be a problem)
  - another run of RAT tests
    - thank you to everyone checking!
  - Tentacles download the jars but expand, but do not preserve the jar :(
    - May need a script
    - Reach out to Apache Infra? In slack
- What about other automations after the Vote approved? suggestion by @tiagobento 
  - Maybe good time to start looking at it!
  - Automation G and F

---
Recording

https://us06web.zoom.us/rec/share/4Tt6-NfJ3t4vIr0-1kt8u0q9rpwOveeiJkJG7BNE5mHcJW4Opd9T0-AO1SvNJ_Du.5c5U1rjx9_b2uVvh

Passcode: uxY0*Zk9

**Comment by dmitry-buykin** [comment]
Hey Alex @porcelli, I tried to use [dmn vscode extension](https://dist.apache.org/repos/dist/dev/incubator/kie/10.0.0-rc1/incubator-kie-10.0.0-rc1-dmn-vscode-extension.vsix) to load updated DMNs of `1.5` version (edited successfully in `kogito-sandbox`), but these files are not readable. Where I could find correct builds for `vscode` extensions, or these extensions are out of scope in `10.0.x`?

<img width="1073" alt="image" src="https://github.com/user-attachments/assets/fca0cd01-03ac-4ae9-8241-35939351852e">
<img width="1120" alt="image" src="https://github.com/user-attachments/assets/6ab355a2-f47a-4afd-b892-fedf393b24fa">

Loaded correctly in the new editor from sandbox:
<img width="1722" alt="image" src="https://github.com/user-attachments/assets/c40baec9-50a0-46b6-8d60-298727e84e32">

**Comment by porcelli** [comment]
1.5 support will be available once Apache KIE 10 is released. It should be very soon...
